ts/src/exmo.ts

```diff
@@ -737,17 +737,57 @@ export default class exmo extends Exchange {
         //         },
         //     }
         //
+        let marginPairsDict = undefined;
+        if (this.checkRequiredCredentials (false)) {
```

Comment:
@samgermain this logic is wrong, if `(this.checkRequiredCredentials (false)` returns true, it means that the keys are available.